sdk/python/dagger/pyproject.toml

```diff
@@ -4,24 +4,29 @@ build-backend = "hatchling.build"
 
 [project]
 name = "dagger"
-version = "0.0.1"
+version = "0.1.0"
 authors = [
   { name="Sam Alba", email="sam@dagger.io" },
 ]
-description = "Dagger client library"
-#readme = "README.md"
-#license = { file="LICENSE" }
-requires-python = ">=3.7"
+description = "Dagger library"
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = ">=3.10"
 classifiers = [
     "Programming Language :: Python :: 3",
-#    "License :: OSI Approved :: MIT License",
+    "License :: OSI Approved :: Apple Public Source License",
```

Comment:
Should it be Apache2 here?

Reply:
Ooops... great catch! I read it as "**Apache** Public Source License" 😅 I even looked for "Apache 2" specifically in https://pypi.org/classifiers/, found these:

- `License :: OSI Approved :: Apache Software License`
- `License :: OSI Approved :: Apache Public Source License`

And tried to figure out which was it. Thanks!